export.js

```diff
@@ -20,28 +21,50 @@ function grabExcelValues(rows, artifact, objTemplate, customFieldRange) {
     }
 }
 
+function getIndentLevel(str){
+    if (str==undefined){
+        return 0;
+    }
+	let indentLevel = 0;
+	str = str.split(" ")
```

Comment:
formatting got a bit messed up here. Is the white space being trimmed automatically by your text editor?